### PR TITLE
Support Iceberg table comments

### PR DESCRIPTION
**Pull request by djsstarburst**
This commit adds support for Iceberg table comments in both
ORC and Parquet formats.    In addition to supporting comments
in table definitions, this commit also support table comment
updates, e.g., COMMENT ON TABLE tablename IS 'table comment',
and adds unit tests for creating, updating and removing
table comments.

### CODE REVIEW

File: presto-iceberg/src/test/java/io/prestosql/plugin/iceberg/TestIcebergSmoke.java [hunk 1]
```diff
@@ -317,6 +317,36 @@ public void testColumnComments()
         dropTable(session, "test_column_comments");
     }
 
+    @Test
+    public void testTableComments()
+    {
+        testTableCommentsForFormat(FileFormat.ORC);
```
Comment (electrum) [inline presto-iceberg/src/test/java/io/prestosql/plugin/iceberg/TestIcebergSmoke.java:323]: We don't need to test multiple formats here, as table comment is only in the metadata, not related to file formats.

Comment (djsstarburst) [reply presto-iceberg/src/test/java/io/prestosql/plugin/iceberg/TestIcebergSmoke.java]: I changed the test to only test the ORC format.  I also added support and tests for COMMENT ON TABLE tablename IS (comment|null).

**Comment by electrum** [comment]
There is a test failure
```
[ERROR] testCommentTable(io.prestosql.plugin.iceberg.TestIcebergDistributed)  Time elapsed: 0.043 s  <<< FAILURE!
java.lang.AssertionError: Expected query to fail: COMMENT ON TABLE orders IS 'hello'
	at org.testng.Assert.fail(Assert.java:94)
	at io.prestosql.testing.QueryAssertions.assertQueryFails(QueryAssertions.java:291)
	at io.prestosql.testing.AbstractTestQueryFramework.assertQueryFails(AbstractTestQueryFramework.java:221)
	at io.prestosql.plugin.iceberg.TestIcebergDistributed.testCommentTable(TestIcebergDistributed.java:58)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
```
Looks like we need to remove that overridden test.

**Comment by djsstarburst** [comment]
LOL @electrum - - I removed the failing test, and ran all the tests in presto-iceberg before pushing.  Should be fixed now.

**Comment by electrum** [comment]
Thanks!
